Model/Ofeq.js

```diff
@@ -1,8 +1,28 @@
-import Satelite from "./Satelite";
```

Comment:
No problem fixing your code, but use rebase to squash it into one commit.
That way I can immediately see what you are doing, and don't need to deal with the errors you have already fixed.